src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperContent.cs

```diff
@@ -48,6 +49,29 @@ public abstract class TagHelperContent : IEnumerable<string>
         public abstract TagHelperContent Append(string value);
 
         /// <summary>
+        /// Appends formatted string by replacing each format item in the specified string 
+        /// with the text equivalent of a corresponding object's value.
```

Comment:
nit: bit roundabout and passive.  suggest using the `string.Format()` doc string as inspiration e.g.
``` c#
/// Appends the specified <paramref name="format"/> to the existing content after
/// replacing each format item with the <see cref="string"/> representation of the
/// corresponding item in the <paramref name="args"/> array.
```